models/MessageShop.js

```diff
@@ -13,46 +13,36 @@ const HospitalSchema = new mongoose.Schema({
         type : String,
         required : [true, 'Please add an address']
     },
-    district: {
+    tel: {
         type : String,
-        required : [true, 'Please type a district']
     },
-    province : {
+    opentime: {
         type : String,
-        required : [true, 'Please type a province']
+        required : [true, 'Please type an open-time']
     },
-    postalcode : {
+    closetime: {
         type : String,
-        required : [true, 'Please type a postalcode'],
-        maxlength : [5, 'Postal Code can not be more than 5 digits']
-    },
-    tel : {
-        type : String,
-    },
-    region : {
-        type : String,
-        required : [true, 'Please type a region']
+        required : [true, 'Please type an close-time']
     }},{
         toJSON : {virtuals : true},
         toObject : {virtuals : true}
     }
 );
 
 //Reverse populate with virtuals
-
-HospitalSchema.virtual('appointments', {
+MessageShopSchema.virtual('appointments', {
     ref : 'Appointment',
```

Comment:
Maybe this line should be ref : 'Reservation' and .virtual('reservations')

Reply:
ok

Reply:
oh I already changed it in another commit ( change from appointment to reservation )